apps/src/music/blockly/blocks/simple2.js

```diff
@@ -0,0 +1,115 @@
+import {BlockTypes} from '../blockTypes';
+import Globals from '../../globals';
+
+// Examine chain of parents to see if one is 'when_run'.
+const isBlockInsideWhenRun = ctx => {
+  // Since this model doesn't currently support triggered code, then
+  // everything is ultimately called from under when_run.
+  return true;
+};
+
+export const whenRunSimple2 = {
+  definition: {
+    type: BlockTypes.WHEN_RUN_SIMPLE2,
+    message0: 'when run',
+    inputsInline: true,
+    nextStatement: null,
+    colour: 230,
```

Comment:
Could we use `style: 'setup_blocks'` for blocks like this and avoid hard-coded color values?

Styles are currently defined in `blockly/addons/cdodTheme.js`

Reply:
Great call; done.